bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -1944,6 +1946,13 @@ private void schedulePowerOnJob() {
                                 sendCommand(RotelCommand.SPEAKER);
                                 Thread.sleep(SLEEP_INTV);
                             }
+                            if (model != RotelModel.RAP1580 && model != RotelModel.RSP1576
+                                    && model != RotelModel.RSP1582) {
```

Comment:
What is special for these three devices - no support for getting model/version? If this is the case, do you know that all other receivers support this - and if not, what will be the impact of sending and waiting for these commands?

Reply:
Yes, the API documentation for these models do not mention these commands. It could be a simple miss in the documentation but I prefer to respect the Rotel documentation.
Of course, I checked the documentation for all other models and these commands are there.
I have no idea what could be the result of sending these commands if not supported. Maybe they will be simply ignored or maybe an error could be returned, I don't know. That's why I prefer to not send the command.